app/controllers/cooks_controller.rb

```diff
@@ -0,0 +1,74 @@
+class CooksController < ApplicationController
+  before_action :set_cook, only: [:show, :edit, :update, :destroy]
+
+  # GET /cooks
```

Comment:
Can you get rid of all the comments? They just clutter the place up.

Reply:
Deffo!